### PR TITLE
Quickstart-wordsmithing

### PR DESCRIPTION
**Pull request by creator-of-data-assurance-hub**

### CODE REVIEW

**Review by leflambeur — CHANGES_REQUESTED** [review]
L25 - I wasn't happy with what was already there but I'm also not sure that 'there's one key concept to shape your thinking' is the answer- we need a plainer/straightforward/less corporate way to say this I think - not sure what would make sense instead I'll think on it

L34 - 'in RKVST **it is** considered an Asset'

L36 - I think this should be restored as the reason it said `an identity, a description` is that it's an almost direct technical translation of the API (so makes the whole thing flow more into the technical concepts)

L60 - Instead of 'decisions about what should happen to it.' it should be'**or if there is a decision made about it**'

L77 - Both versions missed a 'what' in front of Asset

L107 - 'Events on Assets track that build a complete traceable historical record that builds trust in the key information needed by data users.' should be 'Events on Assets which builds a complete and traceable historical record; building trust in the information required by data consumers.

L111 - 'organisations' needs to be Americanized; I'm not sure we need the last sentence in L111

File: content/docs/quickstart/when-who-did-what-to-a-thing/index.md [hunk 1]
```diff
@@ -97,20 +97,20 @@ You can also mix and match ABAC and OBAC into the same policy, for example:
...
+Here we can see an access policy that allows specific members of the Internal Organization, and an External Organisation to read all Event attributes as well as the Height, Width and Length of Assets. The Users can also create "Dispose" Events as well but they cannot cannot change the attribute of an Asset itself.
 
-We can then define filters for the asset types users are allowed to see, which will be [covered in more detail in the IAM Policies Section.]()
+We can then define filters for the Asset types users are allowed to see, which will be [covered in more detail in the IAM Policies Section.]()
 
 The Golden Thread
 -----------------
 
-Now we understand "Who did What to a Thing" we can bring it all together, creating a stream of untamperable, assured and trusted events against an asset to track and follow the key information that is needed. 
+Now we understand "Who did What to a Thing" we can bring it all together, creating a stream of untamperable, assured Events on Assets track that build a complete traceable historical record that builds trust in the key information needed by data users.
```
Comment (JAG-UK) [inline content/docs/quickstart/when-who-did-what-to-a-thing/index.md:107]: "track that build" ?

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: content/docs/quickstart/when-who-did-what-to-a-thing/index.md [hunk 2]
```diff
@@ -31,61 +31,61 @@ Before we begin with RKVST we need to understand one key piece of information:
 What is a Thing?
 ----------------
 
-A thing can be anything; a weather station sensor, an invoice or even a shipping container. Whatever this thing may be, in RKVST it would be considered ***An Asset***.
+A Thing can be anything; a weather station sensor, an invoice or even a shipping container. Whatever this Thing may be, in RKVST considered it an ***Asset***.
```
[the remark this answers is not in the document]
Comment (creator-of-data-assurance-hub) [reply content/docs/quickstart/when-who-did-what-to-a-thing/index.md:34]: fixed

File: content/docs/quickstart/when-who-did-what-to-a-thing/index.md [hunk 3]
```diff
@@ -31,61 +31,61 @@ Before we begin with RKVST we need to understand one key piece of information:
...
 
-What can happen to an Asset?
+What happens to an Asset?
 -----------------------------
 
-Over the lifecycle of an Asset many actions may happen against it, say if an Asset Attribute is changed or there is a noteworthy decision against an Asset, RKVST considers this ***An Event***.
+Many things may happen to an Asset over its entire lifecycle, either directly affecting its state or decisions about what should happen to it. If an Asset Attribute is changed or there is a noteworthy decision upon an Asset, RKVST considers this an ***Event***.
 
-Similarly to an Asset, Events also have an identity, a description and their own set of attributes. Importantly a key attribute of Events is tracking both ***when*** an action happens, ***what*** that action affects and ***who*** acted upon it.
+Similar to an Asset, Events also have identities, descriptions and their own set of attributes. Importantly a key attribute of Events is tracking both ***when*** an action happens, ***what*** that action is and ***who*** acted upon it.
```
[the remark this answers is not in the document]
Comment (creator-of-data-assurance-hub) [reply content/docs/quickstart/when-who-did-what-to-a-thing/index.md]: agree, fixed

**Review by leflambeur — APPROVED** [review]
LGTM

I'm roughly happy with this some minor changes are needed but I can pick those up in my next edit @JAG-UK I'll approve but feel to merge when you're happy

**Pull request merged**